src/pages/Article.tsx

```diff
@@ -0,0 +1,60 @@
+import { Link, useNavigate, useParams } from "react-router-dom";
+import Back from "../components/Back";
+import { useGetFeaturedQuery } from "../features/api/apiSlice";
+
+export default function Article() {
+  const { articleId } = useParams();
+  const navigate = useNavigate();
+
+  const { data: articles } = useGetFeaturedQuery(undefined);
+  const article = articles?.find((article) => article.id === articleId);
```

Comment:
?

Reply:
The articles can be undefined if the state of the query is loading.